src/app/components/NavBar/index.tsx

```diff
@@ -63,14 +63,10 @@ export const Dropdown: React.FunctionComponent<{user: User, currentPath: string}
 };
 
 // tslint:disable-next-line:variable-name
-const DropdownToggle: SFC<{user: User}> = ({user}) =>
-  <FormattedMessage id='i18n:nav:hello:text' values={{name: user.firstName}}>
-    {(msg) => <Styled.DropdownToggle tabIndex='0' data-testid='user-nav-toggle'>
-      {msg}
-      <Styled.DownIcon aria-hidden='true' />
-      <Styled.HamburgerIcon ariaHidden='true' />
-    </Styled.DropdownToggle>}
-  </FormattedMessage>;
+const DropdownToggle: SFC<{user: User}> = ({user: { firstName }}) => {
+  const initials = (firstName[0] + firstName.slice(-1)).toUpperCase();
```

Comment:
https://openstax.slack.com/archives/C8V1XET1R/p1641918768045500?thread_ts=1641918206.044000&cid=C8V1XET1R

> 8px below so that there's enough tap room